src/azure-cli/azure/cli/command_modules/synapse/operations/sqlpool.py

```diff
@@ -27,3 +28,139 @@ def update_sql_pool(cmd, client, resource_group_name, workspace_name, sql_pool_n
+                'jdbc:sqlserver://{workspace_fqdn}:1433;database={sql_pool};'
+                'encrypt=true;trustServerCertificate=false;'
+                'hostNameInCertificate=*{workspace_suffix};loginTimeout=30;Authentication=ActiveDirectoryIntegrated',
+        },
+        SqlPoolConnectionClientType.Odbc: {
+            SqlPoolConnectionClientAuthenticationType.SqlPassword:
+                'Driver={{ODBC Driver 13 for SQL Server}};Server=tcp:{workspace_fqdn},1433;'
+                'Database={sql_pool};Uid=<username>;Pwd=<password>;Encrypt=yes;'
+                'TrustServerCertificate=no;Connection Timeout=30;',
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryPassword:
+                'Driver={{ODBC Driver 13 for SQL Server}};Server=tcp:{workspace_fqdn},1433;'
+                'Database={sql_pool};Uid=<username>;Pwd=<password>;Encrypt=yes;'
+                'TrustServerCertificate=no;Connection Timeout=30;Authentication=ActiveDirectoryPassword',
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryIntegrated:
+                'Driver={{ODBC Driver 13 for SQL Server}};Server=tcp:{workspace_fqdn},1433;'
+                'Database={sql_pool};Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;'
+                'Authentication=ActiveDirectoryIntegrated',
+        },
+        SqlPoolConnectionClientType.Php: {
+            # pylint: disable=line-too-long
+            SqlPoolConnectionClientAuthenticationType.SqlPassword:
+                '$connectionOptions = array("UID"=>"<username>@{workspace}", "PWD"=>"<password>", "Database"=>{sql_pool}, "LoginTimeout" => 30, "Encrypt" => 1, "TrustServerCertificate" => 0); $serverName = "tcp:{workspace_fqdn},1433"; $conn = sqlsrv_connect($serverName, $connectionOptions);',
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryPassword:
+                CLIError('PHP sqlsrv driver only supports SQL Password authentication.'),
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryIntegrated:
+                CLIError('PHP sqlsrv driver only supports SQL Password authentication.'),
+        },
+        SqlPoolConnectionClientType.PhpPdo: {
+            # pylint: disable=line-too-long
+            SqlPoolConnectionClientAuthenticationType.SqlPassword:
+                '$conn = new PDO("sqlsrv:server = tcp:{workspace_fqdn},1433; Database = {sql_pool}; LoginTimeout = 30; Encrypt = 1; TrustServerCertificate = 0;", "<username>", "<password>");',
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryPassword:
+                CLIError('PHP Data Object (PDO) driver only supports SQL Password authentication.'),
+            SqlPoolConnectionClientAuthenticationType.ActiveDirectoryIntegrated:
+                CLIError('PHP Data Object (PDO) driver only supports SQL Password authentication.'),
+        }
+    }
+
+    f = formats[client_provider][auth_type]
```

Comment:
>f [](start = 4, length = 1)

Better to use a more meaningful variable name

Reply:
I agree with you but I am afraid that I can not modify this, because the sql db is using this parameter.  

We hope that sql dw's customer can have the same or similar experience when they use synapse, so I think we should keep consistent with sql db.

If I modify the parameter name here, then I need to add a option list in parameter to expose the same parameter with sql db. I prefer just keep this. @idear1203 How do you like?

---
In reply to: [533113490](https://github.com/Azure/azure-cli/pull/16095#discussion_r533113490) [](ancestors = 533113490)